physionet-django/project/projectfiles/local.py

```diff
@@ -120,7 +121,13 @@ def publish_rollback(self, active_project, published_project):
         os.rename(published_project.file_root(), active_project.file_root())
 
     def publish_complete(self, active_project, published_project):
-        pass
+        if settings.DEFAULT_PROJECT_ACCESS_MECHANISM == 'direct':
+            DataSource.objects.create(
+                project=published_project,
+                files_available=True,
+                data_location=DataSource.DataLocation.DIRECT,
+            )
+
```

Comment:
Same comment as with the GCS files. We should have a simple hook after the project that would call into a class/module to do this. We want this to be as abstract as possible.